<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lockjaw Visualiser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/textures/seal.png</normaloff>:/textures/seal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GLArea" name="glarea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>781</width>
        <height>551</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSlider" name="slider_time">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="commandLinkButton_play_pause">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>pause</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/textures/pause.png</normaloff>:/textures/pause.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="minimumSize">
               <size>
                <width>43</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>43</width>
                <height>23</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="commandLinkButton_start">
              <property name="minimumSize">
               <size>
                <width>71</width>
                <height>41</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>71</width>
                <height>41</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>start</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/textures/play.png</normaloff>:/textures/play.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="minimumSize">
               <size>
                <width>43</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>43</width>
                <height>23</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QDial" name="dial_volume">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>64</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="minimumSize">
               <size>
                <width>118</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>118</width>
                <height>23</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="commandLinkButton_stop">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>stop</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/textures/stop.png</normaloff>:/textures/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionDynamic_Backgroud"/>
    <addaction name="actionFull_Screen"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionDynamic_Backgroud">
   <property name="text">
    <string>Dynamic Backgroud</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLArea</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glarea.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
